src/js/components/DeploymentComponent.jsx

```diff
@@ -97,16 +96,16 @@ var DeploymentComponent = React.createClass({
         </td>
         <td>
           <ul className="list-unstyled">
-            {lazy(model.currentActions).map(function (action) {
+            {model.currentActions.map(function (action) {
               return <li key={action.app}>{action.app}</li>;
-            }).value()}
+            })}
```

Comment:
Generally I tend to agree with @rcorral but I don't mind it in case of oneliners like these – I'd rather not introduce an extra variable for this.